imports/server/users.ts

```diff
@@ -1,8 +1,25 @@
+import { Accounts } from 'meteor/accounts-base';
 import { check } from 'meteor/check';
 import { Meteor } from 'meteor/meteor';
 import { userIdIsAdmin } from '../lib/is-admin';
 import MeteorUsers from '../lib/models/MeteorUsers';
 import { userMaySeeUserInfoForHunt } from '../lib/permission_stubs';
+import { ProfileFields } from '../lib/schemas/User';
+
+const profileFields: Record<ProfileFields, 1> = {
+  displayName: 1,
+  googleAccount: 1,
+  discordAccount: 1,
+  phoneNumber: 1,
+  muteApplause: 1,
+  dingwords: 1,
+};
+
+Accounts.setDefaultPublishFields({
```

Comment:
Maybe worth a comment here that this only changes what fields are autopublished to the `Meteor.user()` object for the logged-in user.